.github/workflows/tests.yaml

```diff
@@ -36,6 +36,8 @@ jobs:
      - name: Test
        env:
         HF_TEST_TOKEN: ${{ secrets.HF_TEST_TOKEN }}
+        HF_HOME: "cache/models"
+        HF_DATASETS_CACHE: "cache/datasets"
```

Comment:
Why did you need to set these?

Reply:
I had errors in the tests but only in the CI, when printing I realized that setting those env variable in the `main_test` file did not work and we were using `/scratch`.